### DISCUSSION TITLE
Mice RNA-seq pathway analysis

### DISCUSSION THREAD

**Discussion by samanehgholami** [egeulgen/pathfindR#136] [General]
Can I use this package for pathway enrichment analysis of Mice RNA-seq data? 
I am new to RNA-seq analysis in Mice! I am searching for a package works in R (with KEGG mice or others), rather than online pathways enrichments and multiple copy, paste and finally analysis in Prism!
I will appreciate if you help me! 
Thank you

**Comment by egeulgen** [comment]
Hello,

Yes, you can use pathfindR for mice data or any other organism. See [this vignette](https://egeulgen.github.io/pathfindR/articles/non_hs_analysis.html)

Particulary see [here](https://egeulgen.github.io/pathfindR/articles/non_hs_analysis.html#built-in-mus-musculus-data). You can use the following to analyze the data:

```r
myeloma_output <- run_pathfindR(input = input_df,
                                convert2alias = FALSE,
                                gene_sets = "mmu_KEGG",
                                pin_name_path = "mmu_STRING")
```

**Reply by samanehgholami** [reply]
Hello,
Thank you very much. This helps me.
I will try this and will ask you if I need more help.

Thank you again.

**Comment by samanehgholami** [comment]
Hello,

I try to the package for mice, but i got these two errors even when i just want to run a sample:

> gsets_list <- get_gene_sets_list(source = "KEGG",
+                                  org_code = "mmu")
 **Error in loadNamespace(x) : there is no package called ‘KEGGREST’** 
> myeloma_output <- run_pathfindR(input = myeloma_input,
+                                 convert2alias = FALSE,
+                                 gene_sets = "Custom",
+                                 custom_genes = mmu_kegg_genes,
+                                 custom_descriptions = mmu_kegg_descriptions,
+                                 pin_name_path = path2SIF)
 **Error in pin_name_path %in% valid_opts[-length(valid_opts)] : 
object 'path2SIF' not found**

Please could you help me with this? I am sorry for asking many questions.

Thank you very much

**Comment by egeulgen** [comment]
You do not have to obtain the data manually, you can run the built-in data via:

```r
output_df <- run_pathfindR(input = input_df,
                                convert2alias = FALSE,
                                gene_sets = "mmu_KEGG",
                                pin_name_path = "mmu_STRING")
```

**Reply by samanehgholami** [reply]
Ok thank you. Now it is working.

**Reply by samanehgholami** [reply]
Hello,

I try to use the list of gene I have and analyze them with pathfindR. I am really thankful for all your help.
I could run the samples you mentioned in package and all other articles you wrote and they work perfect.
But I can not analyze my data, a data frame of gene and adjusted P values.

> sapply(upreg, class)
Gene.symbol       logFC   adj.P.Val 
"character"   "numeric"   "numeric" 
> upreg_output <- run_pathfindR(upreg)
## Testing input
**Error in pathfindR::input_testing(input, p_val_threshold) : 
  p values must all be numeric**

I got this error all the time that p values should be numeric! but they are. I changed them to numeric and as you see it is numeric. But I dont know why I get this.

I do appreciate if you help me again too. thank you

**Reply by egeulgen** [reply]
Can you try `upreg<-as.data.frame(upreg )`, tibbles sometimes cause an issue

**Comment by samanehgholami** [comment]
Dear EgeUlgen,
I also can send the data I have, if it can help. As i mentioned it is on mice but also we may need to check these genes in human pathways too.

Many thanks in advance.
Sama
